libraries/wasm-jit/Source/Platform/POSIX.cpp

```diff
@@ -158,7 +173,11 @@ namespace Platform
 		}
 	}
 
-	THREAD_LOCAL jmp_buf signalReturnEnv;
+	#ifdef __FreeBSD__
```

Comment:
This might actually be a defect in the original WAVM code. Seems like siglongjmp() ought to take a sigjmp_buf on any POSIX (including linux). Should be investigated.

Reply:
This is the error I get without it: https://gist.github.com/zrts/1272e31b5c3699d82fa778de8990e74e

Reply:
Yeah, sorry if I wasn't very clear. What I mean is that I think
`THREAD_LOCAL sigjmp_buf signalReturnEnv;`
Should be correct for Linux/BSD/OSX. We may not need this #ifdef

Reply:
I agree. I don't know why gcc doesn't complain about it though.